### PR TITLE
Feat/data processing

### PR DESCRIPTION
**Pull request by Hugoch**
Add API routes for new beta Data Processing product

### CODE REVIEW

File: src/api/cloud/project/cloud-project.service.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ angular.module('ovh-api-services').service('OvhApiCloudProject', ($injector, $ca
     Acl() {
       return $injector.get('OvhApiCloudProjectAcl');
     },
+    Dataprocessing() {
```
Comment (antleblanc) [inline src/api/cloud/project/cloud-project.service.js]:
```suggestion
    DataProcessing() {
```

File: src/api/cloud/project/dataprocessing/cloud-project-dataprocessing.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectDataProcessingV6', ($resource) => $resource('/cloud/project/:serviceName/dataprocessing', {
```
Comment (antleblanc) [inline src/api/cloud/project/dataprocessing/cloud-project-dataprocessing.v6.service.js]:
```suggestion
  .service('OvhApiCloudProjectDataProcessingV6', ($resource) => $resource('/cloud/project/:serviceName/dataProcessing', {
```

File: src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectDataProcessingAuthorizationIceberg', (iceberg) => iceberg('/cloud/project/:serviceName/dataProcessing/authorization', {
+    serviceName: '@serviceName',
+  }, {
+    post: {
+      method: 'POST',
+      url: '/cloud/project/:serviceName/dataProcessing/authorization',
+      interceptor: postInterceptor,
+      transformResponse: [],
```
Comment (antleblanc) [inline src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js]: Is `transformResponse` still needed?

File: src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  }, {
+    post: {
+      method: 'POST',
+      url: '/cloud/project/:serviceName/dataProcessing/jobs',
+      interceptor: postInterceptor,
+    },
+    delete: {
+      method: 'DELETE',
+      url: '/cloud/project/:serviceName/dataProcessing/jobs/:jobId',
+      transformResponse: [],
```
Comment (antleblanc) [inline src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js]: Is `transformResponse` still needed?

Comment (Hugoch) [reply src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js]: Yes, we return an HTTP 204 (no content) but angular expects a json payload (as stated by default transformResponse). So here we specifically tell angular not to run any transformer on the response.

File: src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js (hunk 3 above)
Comment (Hugoch) [reply src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js]: Yes, we return an HTTP 204 (no content) but angular expects a json payload (as stated by default transformResponse). So here we specifically tell angular not to run any transformer on the response.

Comment (antleblanc) [reply src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js]: Thanks for the explanation.

File: src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js (hunk 4 above)
Comment (antleblanc) [reply src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js]: Thanks for the explanation.

**Pull request merged**